python-3.10/Dockerfile

```diff
@@ -105,30 +105,29 @@ RUN mkdir /home/default \
         sympy \
         tabulate \
         tomlkit \
-        torch \
         tqdm \
         vincent \
         wheel \
         xgboost \
         xlrd \
-	&& chown :users -R /home/default \
-	&& chmod a+rwx -R /home/default
-
-RUN pip3 install --no-cache-dir --upgrade --force-reinstall git+https://github.com/keboola/python-docker-application.git@2.2.0 \
+    && pip3 install --no-cache-dir --upgrade --force-reinstall git+https://github.com/keboola/python-docker-application.git@2.2.0 \
     && pip3 install --no-cache-dir --find-links https://h2o-release.s3.amazonaws.com/h2o/latest_stable_Py.html h2o \
     && pip3 install --no-cache-dir --upgrade --force-reinstall \
         git+https://github.com/keboola/sapi-python-client.git@0.4.0 \
         keboola.component \
-        cryptography\<37 \
+        charset-normalizer\<3 \
+        cryptography\<41 \
+        pytz\<2023 \
+    && pip3 install torch==2.0.0+cpu -f https://download.pytorch.org/whl/torch_stable.html \
```

Comment:
@pepamartinec here's the torch install

Reply:
aha, and why it was nessecary to move it? because it's installed from download.pytorch.org?

Reply:
yeah, also since it's "special" I wanted to give it its own command